### ISSUE TITLE
no noise, only clipping gradient norm

### ISSUE THREAD

**Issue by AlphaPav** [pytorch/opacus#115]
Dear authors, 
I want to clip the norm of the gradient calculated for **each** example in a large dataset and I **don't add noise** after clipping.
How can I leverage this library to achieve that? 
It seems that the `noise_multiplier` in `PrivacyEngine` is always larger than zero in your examples. 
Can I set `noise_multiplier=0`  if I don't add noise after clipping?

**Comment by karthikprasad** [comment]
Hi @AlphaPav, yes; you can set `noise_multiplier` to 0. One of our tests does this: https://github.com/pytorch/opacus/blob/master/opacus/tests/privacy_engine_test.py#L272

Our examples have a non-zero noise as it is required for privacy.
